--- login-update-profile.ftl	2022-02-17 19:25:37.156309500 +0100
+++ login-update-profile-patched.ftl	2022-02-18 23:50:01.343740200 +0100
@@ -1,5 +1,5 @@
 <#import "template.ftl" as layout>
-<@layout.registrationLayout displayMessage=!messagesPerField.existsError('username','email','firstName','lastName'); section>
+<@layout.registrationLayout displayMessage=!messagesPerField.existsError('username','email'); section>
     <#if section = "header">
         ${msg("loginProfileTitle")}
     <#elseif section = "form">
@@ -41,12 +41,12 @@
                 </div>
             </div>
 
-            <div class="${properties.kcFormGroupClass!}">
+            <div class="${properties.kcFormGroupClass!}" style="display:none;">
                 <div class="${properties.kcLabelWrapperClass!}">
                     <label for="firstName" class="${properties.kcLabelClass!}">${msg("firstName")}</label>
                 </div>
                 <div class="${properties.kcInputWrapperClass!}">
-                    <input type="text" id="firstName" name="firstName" value="${(user.firstName!'')}"
+                    <input type="text" id="firstName" name="firstName" value="${(user.firstName!'/')}"
                            class="${properties.kcInputClass!}"
                            aria-invalid="<#if messagesPerField.existsError('firstName')>true</#if>"
                     />
@@ -59,12 +59,12 @@
                 </div>
             </div>
 
-            <div class="${properties.kcFormGroupClass!}">
+            <div class="${properties.kcFormGroupClass!}" style="display:none;">
                 <div class="${properties.kcLabelWrapperClass!}">
                     <label for="lastName" class="${properties.kcLabelClass!}">${msg("lastName")}</label>
                 </div>
                 <div class="${properties.kcInputWrapperClass!}">
-                    <input type="text" id="lastName" name="lastName" value="${(user.lastName!'')}"
+                    <input type="text" id="lastName" name="lastName" value="${(user.lastName!'/')}"
                            class="${properties.kcInputClass!}"
                            aria-invalid="<#if messagesPerField.existsError('lastName')>true</#if>"
                     />
