--- AccountPage.tsx	2022-02-17 19:38:28.036309500 +0100
+++ AccountPage-patched.tsx	2022-02-17 19:37:38.256309500 +0100
@@ -35,8 +35,6 @@
 
 interface FormFields {
     readonly username?: string;
-    readonly firstName?: string;
-    readonly lastName?: string;
     readonly email?: string;
     attributes?: { locale?: [string] };
 }
@@ -58,14 +56,10 @@
     private readonly DEFAULT_STATE: AccountPageState = {
         errors: {
             username: '',
-            firstName: '',
-            lastName: '',
             email: ''
         },
         formFields: {
             username: '',
-            firstName: '',
-            lastName: '',
             email: '',
             attributes: {}
         }
@@ -178,44 +172,6 @@
                         >
                         </TextInput>
                     </FormGroup>
-                    <FormGroup
-                        label={Msg.localize('firstName')}
-                        isRequired
-                        fieldId="first-name"
-                        helperTextInvalid={this.state.errors.firstName}
-                        isValid={this.state.errors.firstName === ''}
-                    >
-                        <TextInput
-                            isRequired
-                            type="text"
-                            id="first-name"
-                            name="firstName"
-                            maxLength={254}
-                            value={fields.firstName}
-                            onChange={this.handleChange}
-                            isValid={this.state.errors.firstName === ''}
-                        >
-                        </TextInput>
-                    </FormGroup>
-                    <FormGroup
-                        label={Msg.localize('lastName')}
-                        isRequired
-                        fieldId="last-name"
-                        helperTextInvalid={this.state.errors.lastName}
-                        isValid={this.state.errors.lastName === ''}
-                    >
-                        <TextInput
-                            isRequired
-                            type="text"
-                            id="last-name"
-                            name="lastName"
-                            maxLength={254}
-                            value={fields.lastName}
-                            onChange={this.handleChange}
-                            isValid={this.state.errors.lastName === ''}
-                        >
-                        </TextInput>
-                    </FormGroup>
                     {features.isInternationalizationEnabled && <FormGroup
                         label={Msg.localize('selectLocale')}
                         isRequired
