--- AccountPage.tsx 2022-04-27 00:00:37.834438400 +0200
+++ AccountPage-patched.tsx 2022-04-26 23:58:38.784438400 +0200
@@ -49,8 +49,6 @@
 
 interface FormFields {
     readonly username?: string;
-    readonly firstName?: string;
-    readonly lastName?: string;
     readonly email?: string;
     attributes?: { locale?: [string] };
 }
@@ -72,14 +70,10 @@
     private readonly DEFAULT_STATE: AccountPageState = {
         errors: {
             username: '',
-            firstName: '',
-            lastName: '',
             email: ''
         },
         formFields: {
             username: '',
-            firstName: '',
-            lastName: '',
             email: '',
             attributes: {}
         }
@@ -212,56 +206,6 @@
                                         ? ValidatedOptions.error
                                         : ValidatedOptions.default
                                 }
-                            ></TextInput>
-                        </FormGroup>
-                        <FormGroup
-                            label={Msg.localize("firstName")}
-                            fieldId="first-name"
-                            helperTextInvalid={this.state.errors.firstName}
-                            validated={
-                                this.state.errors.firstName !== ""
-                                    ? ValidatedOptions.error
-                                    : ValidatedOptions.default
-                            }
-                        >
-                            <TextInput
-                                isRequired
-                                type="text"
-                                id="first-name"
-                                name="firstName"
-                                maxLength={254}
-                                value={fields.firstName}
-                                onChange={this.handleChange}
-                                validated={
-                                    this.state.errors.firstName !== ""
-                                        ? ValidatedOptions.error
-                                        : ValidatedOptions.default
-                                }
-                            ></TextInput>
-                        </FormGroup>
-                        <FormGroup
-                            label={Msg.localize("lastName")}
-                            fieldId="last-name"
-                            helperTextInvalid={this.state.errors.lastName}
-                            validated={
-                                this.state.errors.lastName !== ""
-                                    ? ValidatedOptions.error
-                                    : ValidatedOptions.default
-                            }
-                        >
-                            <TextInput
-                                isRequired
-                                type="text"
-                                id="last-name"
-                                name="lastName"
-                                maxLength={254}
-                                value={fields.lastName}
-                                onChange={this.handleChange}
-                                validated={
-                                    this.state.errors.lastName !== ""
-                                        ? ValidatedOptions.error
-                                        : ValidatedOptions.default
-                                }
                             ></TextInput>
                         </FormGroup>
                         {features.isInternationalizationEnabled && (
